src/PensionsRegulatorApi/Security/SecurityServicesCollectionExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
```

Comment:
Not to be done now, but is it worth thinking about moving extensions into a shared package?